stdlib/public/Concurrency/linker-support/magic-symbols-for-install-name.c

```diff
@@ -34,6 +34,11 @@
 // Xcode inserting a runpath search path of /usr/lib/swift based on the deployment target being less than
 // SupportedTargets[target][SwiftConcurrencyMinimumDeploymentTarget] in SDKSettings.plist.
 
+// Clients can back deploy to OS versions that predate Concurrency as an embedded library, and conditionally use
+// it behind an #availability check. Such clients will still need to link the embedded library instead of the OS version.
```

Comment:
Mega nitpick: I should've typed this out in the actual header. Can you try to make the right margin roughly line up with the previous paragraph? My lines were just a little too long.

Reply:
Done.